spec/factories.rb

```diff
@@ -50,6 +50,14 @@
     sequence(:change_note) { |n| "Change note #{n}" }
   end
 
+  factory :manual do
+    slug "manual-slug"
+
+    initialize_with do
```

Comment:
No. As I mentioned in the 2nd commit note:

> The factory is a bit non-standard, because `Manual` does not implement
attribute writer methods which is what `FactoryGirl` uses by default to
set the attributes.